js/square.js

```diff
@@ -1,5 +1,12 @@
 define(["js/properties", "js/squareImages", "js/sound"], (properties, squareImages, sound) => {
 
+    function animationCallBack()
```

Comment:
Is there a reason we don't use an arrow function here?

Reply:
I think that was a merge conflict. Is that also a lint issue?

Reply:
No it's not really a linting issue. It's just that we try to use ES6 functions instead of using the old school functions.